File: mobile/src/main/java/net/mm2d/dmsexplorer/log/EmptySender.java
/*
 * Copyright (c) 2018 大前良介 (OHMAE Ryosuke)
 *
 * This software is released under the MIT License.
 * http://opensource.org/licenses/MIT
 */

package net.mm2d.dmsexplorer.log;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;

/**
 * @author <a href="mailto:ryo@mm2d.net">大前良介 (OHMAE Ryosuke)</a>
 */
public class EmptySender implements Sender {
    EmptySender() {
    }

    @Override
    public void logEvent(
            @NonNull final String name,
            @Nullable final Bundle params) {
    }
}

File: mobile/src/main/java/net/mm2d/dmsexplorer/log/EmptySender.kt
/*
 * Copyright (c) 2018 大前良介 (OHMAE Ryosuke)
 *
 * This software is released under the MIT License.
 * http://opensource.org/licenses/MIT
 */

package net.mm2d.dmsexplorer.log

import android.os.Bundle

/**
 * @author [大前良介 (OHMAE Ryosuke)](mailto:ryo@mm2d.net)
 */
internal class EmptySender internal constructor() : Sender {
    override fun logEvent(
        name: String,
        params: Bundle?,
    ) {
    }
}
